### PR TITLE
Birthday events page

### PR DESCRIPTION
**Pull request by GregSutcliffe**
First commit is just sanitising the events layout, there was way too much html code in events/index.html

Second commit adds a birthday page detailing the events so far - comments on each event added inline...

### CODE REVIEW

File: events/birthday.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+The birthday party kicks off in Edinburgh, where our very own community guy,
+[Greg Sutcliffe](https://twitter.com/gwmngilfen) will be giving a Foreman talk
+to the [Techmeetup](http://techmeetup.co.uk) community. This is likely to be an
+overview talk, covering the basics of Foreman, what's new in 1.12, and it's
+awesome community.
+
+This talk should be recorded and livestreamed (links will be added here once we
+know them) so do tune in and get involved.
+
```
Comment (GregSutcliffe) [inline events/birthday.md:38]: @marksutherland is this OK to confirm?

File: events/birthday.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+The awesome folks at NETWAYS are hosting a halfday Foreman event, kicking off
+with 2+ hours of hands-on time with Foreman - perfect for your first steps with
+Foreman, to get help from some of the experienced people present, or just to do
+some in-person hacking.
+
+We'll then break for coffee before having some talks from users and
+contributors, and then head out for some food.
+
+Further links and schedule will be added once it's all confirmed.
+
```
Comment (GregSutcliffe) [inline events/birthday.md:53]: @dgoetz We can add a copy of the schedule here and a link to NETWAYS etc, once it's all finalised - but this should be enough for you to link to in your blog post right?

File: events/birthday.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+### Foreman @ ILTechTalks Israel (Date TBD)
+
+The folks at [ILTechTalks](http://www.meetup.com/ILTechTalks) are working on a
+Foreman event sometime in July - more details to follow!
+
```
Comment (GregSutcliffe) [inline events/birthday.md]: @tbrisker @shlomizadok @imriz any updates on this? Shall I leave it in or would you rather take it out until it's confirmed?

File: events/birthday.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+### Foreman @ Washington DC (Date TBD)
+
+Chris Pisano at The Advisory Board Company is working on hosting
+a Foreman meetup for the end of July, likely to be an evening
+event with some speakers and chat.
```
Comment (GregSutcliffe) [inline events/birthday.md:58]: @discreet is this OK for you or shall I remove it until you can confirm?

Comment (discreet) [inline events/birthday.md:58]: @GregSutcliffe looks good to me. I'll send a PR once I have confirmations.

File: events/birthday.md (hunk 2 above)
Comment (dgoetz) [inline events/birthday.md:53]: Yes, its enough to get promoting it started. In the blogpost I will add all already confirmed details and after all is finalised we can add it here.

**Comment by mmoll** [comment]
looking good for me

File: events/birthday.md (hunk 1 above)
Comment (GregSutcliffe) [inline events/birthday.md:38]: +1 from a private comment received, this section is good to go ;)

File: events/birthday.md (hunk 3 above)
Comment (GregSutcliffe) [inline events/birthday.md]: I'll drop this for now, grapevine says dates should be coming next week

**Comment by GregSutcliffe** [comment]
3 confirmed, 1 retracted for now. Good to go from my side!

**Comment by mmoll** [comment]
thanks @GregSutcliffe!

**Pull request merged**